runtime/laos/src/configs/election_phragmen.rs

```diff
@@ -0,0 +1,60 @@
+use crate::{
+	currency::UNIT, weights, Balance, Balances, BlockNumber, Council, Runtime, RuntimeEvent,
+	Treasury,
+};
+use frame_support::parameter_types;
+#[cfg(not(feature = "fast-mode"))]
+use parachains_common::DAYS;
+#[cfg(feature = "fast-mode")]
+use parachains_common::HOURS;
+use polkadot_runtime_common::CurrencyToVote;
+
+#[cfg(feature = "fast-mode")]
+pub const TERM_DURATION: BlockNumber = 4 * HOURS;
+#[cfg(not(feature = "fast-mode"))]
+pub const TERM_DURATION: BlockNumber = 28 * DAYS;
+
+#[cfg(feature = "fast-mode")]
+pub const ELECTION_VOTING_LOCK_DURATION: BlockNumber = 4 * HOURS;
+#[cfg(not(feature = "fast-mode"))]
+pub const ELECTION_VOTING_LOCK_DURATION: BlockNumber = 28 * DAYS;
+
+parameter_types! {
+	pub const CandidacyBond: Balance = 1000 * UNIT;
+	pub TermDuration: BlockNumber = TERM_DURATION;
+	pub VotingLockPeriod: BlockNumber = ELECTION_VOTING_LOCK_DURATION;
+	pub const DesiredMembers: u32 = 7;
+	pub const DesiredRunnersUp: u32 = 20;
+	pub const MaxCandidates: u32 = 30;
+	pub const MaxVoters: u32 = 200;
+	pub const MaxVotesPerVoter: u32 = 8;
+	pub const VotingBondBase: Balance = 1000 * UNIT;
+	pub const VotingBondFactor: Balance = 100 * UNIT;
+}
+
+impl pallet_elections_phragmen::Config for Runtime {
+	/// How much should be locked up in order to submit one's candidacy.
+	type CandidacyBond = CandidacyBond;
+	type ChangeMembers = Council;
+	type Currency = Balances;
+	type CurrencyToVote = CurrencyToVote;
+	/// Number of members to elect.
+	type DesiredMembers = DesiredMembers;
+	/// Number of runners_up to keep.
+	type DesiredRunnersUp = DesiredRunnersUp;
+	type InitializeMembers = Council;
+	type LoserCandidate = Treasury;
+	type MaxCandidates = MaxCandidates;
+	type MaxVoters = MaxVoters;
+	type MaxVotesPerVoter = MaxVotesPerVoter;
+	type PalletId = ();
```

Comment:
in the rococo runtime they add some id: https://github.com/paritytech/polkadot-sdk/blob/0bb6249268c0b77d2834640b84cb52fdd3d7e860/polkadot/runtime/rococo/src/lib.rs#L1588-L1589
we might want to do the same

Reply:
+1. We've done a similar thing for the treasury pallet -> https://github.com/freeverseio/laos/pull/716/files#diff-094b4a74f7e7a446c70dd2ca710d6a7c87a1c46369ec9a8125621479e247219aR30